lib/tool_belt/issue_service.rb

```diff
@@ -3,28 +3,30 @@
 module ToolBelt
   # Find issues for a particular project & release
   class IssueService
-    attr_accessor :project_name, :release, :redmine_release_id, :prior_releases
+    attr_accessor :project_name, :release, :redmine_version_id, :prior_releases
 
     def initialize(config)
       self.project_name = config.project
       self.release = config.release
-      self.redmine_release_id = config.redmine_release_id
+      self.redmine_version_id = config.redmine_version_id
       self.prior_releases = config.prior_releases || []
     end
 
     def project
       @project ||= Redmine::Project.new(project_name)
     end
 
-    def release_issues(release_id)
-      project.get_issues_for_release(release_id)
+    def release_issues(version_id)
+      project.issues_for_version(version_id)
     end
 
-    def load_issues
-      issues = release_issues(redmine_release_id)
+    def load_issues(include_prior)
```

Comment:
Can we set this to false by default so functions don't have to pass this in?

Reply:
I intended to do that, but clearly didn't.

Reply:
Question is if it should default to true or false. True would be backwards compatible but a user might not expect this and false might make more sense.

Reply:
We also now have the multiple releases way of saying an issue was delivered which simplify/complicates things as well.